web/template/supporter/organisation_details.gohtml

```diff
@@ -20,16 +20,73 @@
 
       <h1 class="govuk-heading-xl">{{ tr .App "manageOrganisation" }}</h1>
 
-      <div class="govuk-tabs app-tabs-no-border">
+      <div class="govuk-tabs app-tabs-no-border" data-module="govuk-tabs">
         <ul class="govuk-tabs__list">
           <li class="govuk-tabs__list-item">
-            <a class="govuk-tabs__tab" href="#">{{ tr .App "manageTeamMembers" }}</a>
+            <a class="govuk-tabs__tab" href="#team-members">{{ tr .App "manageTeamMembers" }}</a>
           </li>
           <li class="govuk-tabs__list-item govuk-tabs__list-item--selected">
-            <a class="govuk-tabs__tab" href="#">{{ tr .App "organisationDetails" }}</a>
+            <a class="govuk-tabs__tab" href="#organisation-details">{{ tr .App "organisationDetails" }}</a>
           </li>
         </ul>
-        <div class="govuk-tabs__panel">
+        <div class="govuk-tabs__panel" id="team-members">
+          {{ if .Query.Has "email" }}
+            <div class="govuk-notification-banner govuk-notification-banner--success" role="alert"
+                 aria-labelledby="govuk-notification-banner-title"
+                 data-module="govuk-notification-banner">
+              <div class="govuk-notification-banner__header">
+                <h1 class="govuk-notification-banner__title" id="govuk-notification-banner-title">
+                  {{ tr .App "invitationSent" }}
+                </h1>
+              </div>
+              <div class="govuk-notification-banner__content">
+                <p class="govuk-body app-full-width">
+                  {{ trFormatHtml .App "weHaveSentInvite" "Email" (.Query.Get "email") }}
+                </p>
+              </div>
+            </div>
+          {{ end}}
+
+          <a class="govuk-button" href="{{ link .App global.Paths.Supporter.InviteMember.Format }}">{{ tr .App "inviteTeamMember" }}</a>
+
+          {{ if .InvitedMembers }}
+            <h2 class="govuk-heading-m">{{ tr .App "invitedTeamMembers" }}</h2>
+
+            <dl class="govuk-summary-list app-summary-list--border-top">
```

Comment:
would a table be better? the last column is taking up a lot of space causing the email to wrap

Reply:
I spent about 15 minutes trying to work out how to space this better for the email and gave up. I think I'd copied the markup from another list of things in the app but hadn't twigged it was for fewer props displayed. I'll switch to a table instead